helm/ffc-demo-payment-service/values.yaml

```diff
@@ -9,6 +9,10 @@ oidcProvider: dev
 okta: {}
 b2c: {}
 
+appInsights:
+  key:
```

Comment:
I think I'd rather there be no key so App Insights is not enabled, rather than a dummy key that appears to work, but does nothing and still communicates with the App Insights end point.

Reply:
Yeah, fair point.